cactusbot/services/beam/liveloading.py

```diff
@@ -34,65 +34,32 @@ def __init__(self, channel, user):
 
         packet = await self.parse(await self.receive())
 
-        asyncio.ensure_future(self.ping(packet["data"]["pingInterval"]/1000))
-
         await super().read(handle)
 
     async def initialize(self, *interfaces):
         """Subscribe to liveloading interfaces."""
 
         if not interfaces:
-            # TODO: dict of interfaces with callbacks
-            interfaces = (
-                "channel:{channel}:update",
-                "channel:{channel}:status",
-                "channel:{channel}:followed",
-                "channel:{channel}:subscribed",
-                "user:{user}:followed",
-                "user:{user}:subscribed",
-                "user:{user}:achievement"
-            )
-
-        interfaces = tuple(
```

Comment:
I like this better - instead of "pre-formatting" strings, we're able to pass an interface with un-formatted strings, and it'll autoformat. Allows for more "dynamic" code, such as the following. (You can just change the passed user/channel, and everything auto-updates.)

```python
constellation.initialize("channel:{channel}:followed")
```